frontend/src/metabase/dashboard/components/DashboardGrid.tsx

```diff
@@ -1,6 +1,7 @@
 import cx from "classnames";
```

Comment:
This is something that took a bit to fix...
So basically a "thunk creator" is a function that returns a function.
Calling `dispatch(thunkCreator(xxx))` though calls the returned function and awaits the relevant promise is present, so `ReturnType<thunkCreator()> != ReturnType<dispatch(thunkCreator()>`.
The easiest way we found of making typescript infer the correct type was to do as suggested on the docs and let typescript infer the prop types by the `mapDispatchToProps` function.
We actually had to add it to `mapDispatchToProps` as it was previously using the one coming from the prop drilling (which we all know is ugly)